.travis.yml

```diff
@@ -9,10 +9,28 @@ stages:
   - test
   - name: deploy
     if: type != pull_request
+before_install:
+  - |
+      if [[ -z "$TRAVIS_COMMIT_RANGE" ]]; then
+          # Builds triggered by initial commit of a new branch.
+          DOCS_ONLY=0
+      else
+          DOCS_REGEX='(OWNERS|LICENSE)|(\.md$)|(^docs/)'
+          [[ -z "$(git diff --name-only $TRAVIS_COMMIT_RANGE | grep -vE $DOCS_REGEX)" ]]
+          DOCS_ONLY=$?
+      fi
 jobs:
   include:
     # Test
-    - script: make verify build build-integration build-e2e test images svcat
+    script:
+    - |
```

Comment:
I think `script` should have a `-` before it. `include` takes a list of stages, and we're trying to say the first stage (whose name defaults to "test") uses the given script.